packages/python/requirements.txt

```diff
@@ -1,6 +1,7 @@
 black==22.3.0
 Django==3.2.14
 Flask==2.0.1
+pylint==2.14.5
```

Comment:
Is there any concept of devDependencies in pip, or is everything just bundled together?